homeassistant/components/zwave_js/climate.py

```diff
@@ -510,17 +510,19 @@ async def async_set_preset_mode(self, preset_mode: str) -> None:
                 "Dry and Fan preset modes are deprecated and will be removed in Home Assistant 2024.2. "
                 "Please use the corresponding Dry and Fan HVAC modes instead"
             )
+            breaks_in_ha_version = "2024.2.0"
             async_create_issue(
                 self.hass,
                 DOMAIN,
                 f"dry_fan_presets_deprecation_{self.entity_id}",
-                breaks_in_ha_version="2024.2.0",
+                breaks_in_ha_version=breaks_in_ha_version,
                 is_fixable=True,
                 is_persistent=True,
                 severity=IssueSeverity.WARNING,
                 translation_key="dry_fan_presets_deprecation",
                 translation_placeholders={
                     "entity_id": self.entity_id,
+                    "breaks_in_ha_version": breaks_in_ha_version,
```

Comment:
We don't need to put the version in there at all.

The issue dialog will already show that. This adds now duplicate information; not sure if that is needed at all?